resteasy-spring-boot-starter-test/pom.xml

```diff
@@ -100,76 +100,76 @@
+<!--                        <groupId>com.fortify.ps.maven.plugin</groupId>-->
+<!--                        <artifactId>sca-maven-plugin</artifactId>-->
+<!--                        <version>4.22</version>-->
+<!--                    </plugin>-->
+<!--                </plugins>-->
+<!--            </build>-->
+<!--        </profile>-->
+<!--    </profiles>-->
 
     <build>
-        <pluginManagement>
-            <plugins>
-                <!-- Added for Fortify Support -->
-                <plugin>
-                    <groupId>com.fortify.ps.maven.plugin</groupId>
-                    <artifactId>sca-maven-plugin</artifactId>
-                    <configuration>
-                        <source>1.8</source>
-                        <maxHeap>1G</maxHeap>
-                    </configuration>
-                    <executions>
-                        <execution>
-                            <id>sca-clean</id>
-                            <goals>
-                                <goal>clean</goal>
-                            </goals>
-                        </execution>
-                        <execution>
-                            <id>sca-translate-exec</id>
-                            <goals>
-                                <goal>translate</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                    <version>4.22</version>
-                </plugin>
-                <!-- End Added For Fortify Support -->
-            </plugins>
-        </pluginManagement>
+<!--        <pluginManagement>-->
+<!--            <plugins>-->
+<!--                &lt;!&ndash; Added for Fortify Support &ndash;&gt;-->
+<!--                <plugin>-->
+<!--                    <groupId>com.fortify.ps.maven.plugin</groupId>-->
+<!--                    <artifactId>sca-maven-plugin</artifactId>-->
+<!--                    <configuration>-->
+<!--                        <source>1.8</source>-->
+<!--                        <maxHeap>1G</maxHeap>-->
+<!--                    </configuration>-->
+<!--                    <executions>-->
+<!--                        <execution>-->
+<!--                            <id>sca-clean</id>-->
+<!--                            <goals>-->
+<!--                                <goal>clean</goal>-->
+<!--                            </goals>-->
+<!--                        </execution>-->
+<!--                        <execution>-->
+<!--                            <id>sca-translate-exec</id>-->
+<!--                            <goals>-->
+<!--                                <goal>translate</goal>-->
+<!--                            </goals>-->
+<!--                        </execution>-->
+<!--                    </executions>-->
+<!--                    <version>4.22</version>-->
+<!--                </plugin>-->
+<!--                &lt;!&ndash; End Added For Fortify Support &ndash;&gt;-->
+<!--            </plugins>-->
+<!--        </pluginManagement>-->
```

Comment:
Same with this, if not using it just delete it :) Unless of course it will be required in the future.

Reply:
yes sir!